core/livepeernode.go

```diff
@@ -87,10 +87,11 @@ type LivepeerNode struct {
 	// Thread safety for config fields
 	mu sync.RWMutex
 	// Transcoder private fields
-	priceInfo     *big.Rat
-	serviceURI    url.URL
-	segmentMutex  *sync.RWMutex
-	StorageConfig *transcodeConfig
+	priceInfo      *big.Rat
+	serviceURI     url.URL
+	segmentMutex   *sync.RWMutex
+	StorageConfigs map[string]*transcodeConfig
```

Comment:
Nit: can't it be package-private, like `storageConfigs`? Do you use it outside of this package?